packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -21,13 +20,10 @@ watch(visibility, (visibilityState) => {
 const showUserStack = computed(() => collaborationStore.collaborators.length > 1);
 
 const collaboratorsSorted = computed(() => {
-	const currentWorkflowUsers = collaborationStore.collaborators.map(({ user }) => user);
-	const owner = currentWorkflowUsers.find(isUserGlobalOwner);
-	return {
-		defaultGroup: owner
-			? [owner, ...currentWorkflowUsers.filter((user) => user.id !== owner.id)]
-			: currentWorkflowUsers,
-	};
+	const users = collaborationStore.collaborators.map(({ user }) => user);
+	const index = users.findIndex((user) => user.id === usersStore.currentUser?.id);
```

Comment:
Should we check if it's not found, i.e. `index === -1`?

Reply:
or, did you mean if the current user is missing in the payload sent over the websocket?

Reply:
Mainly the former, but I guess it could be either case. `.splice` seems to remove from the end if the index is `-1`. Maybe we could add an `assert` or `if` statement just to be sure